draft-ietf-cdni-ci-triggers-rfc8007bis.xml

```diff
@@ -2174,14 +2187,14 @@ uCDN-A                         dCDN-B                         dCDN-C
       <section anchor="time-policy" title="TimePolicy Extension">
         <t>uCDN may wish to perform content management operations on dCDN on a specific schedule.
            The TimePolicy extension allows uCDN to instruct dCDN to execute the trigger command
-           in a desired time window. For example, a content provider that wishes to pre-populate a new
+           in a desired time window. For example, a content provider wishes to pre-populate a new
            episode at off-peak time so that it would be ready on caches at prime time when the episode
            is released for viewing. A scheduled operation enables uCDN to direct dCDN in what
            time frame to execute the trigger.
         </t>
         <t>This specification supports regional scheduling, so when used in conjunction with the Location Policy defined
               in <xref target="location-policy"/>, uCDN can trigger separate commands for
-              different geographical regions, for each region using a different schedule. This allows
+              different geographical regions, for each region, using a different schedule. This allows
```

Comment:
There is a lot packed into this sentence, and I might recommend breaking it up some.  Maybe:
"This specification supports regional scheduling when used in conjunction with the Location Policy defined in <xref target="location-policy"/>.  uCDN can trigger separate commands for different geographical regions using a different schedule for each region.  This allows uCDN to control the execution time per region."

Reply:
maybe the first sentence could read
This specification supports region-by-region time scheduling when used in conjunction with the Location Policy  
defined in <xref target="location-policy"/>.